install/kubernetes/cilium/values.yaml

```diff
@@ -2721,6 +2721,9 @@ clustermesh:
       # -- Security context to be added to clustermesh-apiserver etcd containers
       securityContext: {}
 
+      # -- lifecycle setting for the etcd container.
+      lifecycle: {}
```

Comment:
Please add the the lifecycle configuration also to the kvstoremesh container below.